.github/workflows/platform.dependencies.yml

```diff
@@ -49,7 +49,6 @@ jobs:
       - name: 'Deploy module'
         uses: ./.github/actions/templates/deployModule
         with:
-          moduleName: '${{ env.moduleName }}'
```

Comment:
Should we also remove the `moduleName` env variable in each job as mentioned yesterday?

Reply:
done